impacthub/chain.json

```diff
@@ -35,24 +35,24 @@
   },
   "codebase": {
     "git_repo": "https://github.com/ixofoundation/ixo-blockchain",
-    "recommended_version": "v3.0.0",
+    "recommended_version": "v4.0.0",
```

Comment:
This version isn't anywhere in the versions.json file. Please add a "v4" there.

Reply:
@JeremyParish69 Hey there - resolved in this commit.